### ISSUE TITLE
ce_loss vs dft_loss

### ISSUE THREAD

**Issue by NaOH678** [yongliang-wu/DFT#32]
Hello! Thanks for your excellent work. Is the following loss right? The dft loss is too small?

<img width="707" height="674" alt="Image" src="https://github.com/user-attachments/assets/992970b7-e470-4e8b-8ad5-b158caf0bf86" />

**Comment by NaOH678** [comment]
Could you please share the training loss log（origin loss and dft loss）？ Thank you very much

**Comment by lchen1019** [comment]
The DFT loss is weighted using logits, while the SFT loss is weighted using 1, so the former is generally smaller.
